index.html

```diff
@@ -258,6 +262,60 @@ <h3>Video Content Hints</h3>
       </p></td></tr>
     </tbody></table>
   </section>
+  <section>
+    <h2>Behaviors of other components based on content-hint</h2>
+    <section>
+      <h3>Behavior of a MediaStreamTrack</h3>
+      <p>A MediaStreamTrack for which a contentHint attribute has been
+      set MUST set the following default settings for constrainable
```

Comment:
Do we expect the effect to be synchronously observable on the track?

If so, I'd prefer something like *"On setting of `contentHint`, the user agent must run the following steps"*.

Reply:
That's why we have review - I'll see if I can come up with suitable language.